Tasks/Common/clienttool-common/ClientToolUtilities.ts

```diff
@@ -0,0 +1,198 @@
+import AdmZip = require('adm-zip');
```

Comment:
Looks like most of the code is copied from https://github.com/microsoft/azure-pipelines-tasks/blob/585663dc99b62b5ab75723091d639eaf77bfade6/common-npm-packages/packaging-common-v3/universal/ArtifactToolUtilities.ts . Any reason why we couldn't use this, instead of duplicating the code?